docs/publications.md

```diff
@@ -0,0 +1,79 @@
+## Publications
+The following publications used webKnossos for for exploring large 3D electron-microscopy (EM) and light-microscopy (LM) datasets as well as collaboratively generating annotations or other tasks. Please get in touch with us if your publication is missing.
+
+## 2019
+> Ali Karimi, Jan Odenthal, Florian Drawitsch, Kevin M. Boergens, Moritz Helmstaedter.  
```

Comment:
I would use bullets instead of paragraphs:
```
* Ali Karimi, Jan Odenthal, Florian Drawitsch, Kevin M. Boergens, Moritz Helmstaedter.  
  Cell-type specific innervation of cortical pyramidal cells at their apical tufts.  
  [bioRxiv (2019) DOI:10.1101/571695.](https://www.biorxiv.org/content/10.1101/571695v1.abstract)
```